_posts/2022-02-04-introducing-arrow-flight-sql.md

```diff
@@ -0,0 +1,173 @@
+---
+layout: post
+title: "Introducing Apache Arrow Flight SQL: Accelerating Database Access"
+description: "This post introduces Arrow Flight SQL, a protocol for interacting
+with SQL databases over Arrow Flight. We have been working on this protocol
+over the last six months, and are looking for feedback, interested
+contributors, and early adopters."
+date: "2022-02-04 00:00:00 -0500"
+author: James Duong, Kyle Porter, David Li
```

Comment:
This list needs to be extended.

Reply:
Please add José Almeida, Juscelino Junior, Rafael Telles and Vinicius Fraga